gallery_dl/extractor/blogger.py

```diff
@@ -37,7 +37,8 @@ def items(self):
         findall_image = re.compile(
             r'src="(https?://(?:'
             r'blogger\.googleusercontent\.com/img|'
-            r'lh\d+\.googleusercontent\.com/|'
+            r'lh\d+\.googleusercontent\.com|'
+            r'lh\d+-\w+\.googleusercontent\.com|'
```

Comment:
```suggestion
            r'lh\d+(?:-\w+)?\.googleusercontent\.com|'
```
these two can be combined

Reply:
Thanks! That looks a lot better.